packages/eas-cli/src/build/ios/prepareJob.ts

```diff
@@ -34,7 +34,6 @@ interface CommonJobProperties {
   platform: Platform.IOS;
   projectArchive: ArchiveSource;
   builderEnvironment: Ios.BuilderEnvironment;
-  releaseChannel?: string;
```

Comment:
Never defined at this level. And if it was it would be overwritten here: https://github.com/expo/eas-cli/pull/461/files#diff-22985259d1ba471645af133efec17c58e7a80e3244e5c7644a66f02a4b483899L129